src/dvb.c

```diff
@@ -1864,8 +1865,8 @@ int get_signal_new(adapter *ad, int *status, uint32_t *ber, uint16_t *strength,
         init_snr = enum_cmdargs[1].u.st.stat[0].svalue; // dB * 1000
         *db = (int)get_snr_decibels(init_snr, &snrd, ad->db_snr_map);
     }
-    //	else if (enum_cmdargs[1].u.st.stat[0].scale == 0)
-    //		err |= 2;
+    //	else if (enum_cmdargs[1].u.st.stat[0].scale == FE_SCALE_NOT_AVAILABLE)
```

Comment:
nit: remove the lines as they are mot needed

Reply:
Removed (for both properties).